<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReperMainWindowForm</class>
 <widget class="QMainWindow" name="ReperMainWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_DB">
    <property name="title">
     <string>&amp;DB</string>
    </property>
    <addaction name="actConnect"/>
    <addaction name="actDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actRLI"/>
    <addaction name="separator"/>
    <addaction name="actE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_RLI_Analisys">
    <property name="title">
     <string>&amp;RLI Analisys</string>
    </property>
    <widget class="QMenu" name="menu_Search">
     <property name="title">
      <string>&amp;Search...</string>
     </property>
     <addaction name="actSearchImage"/>
     <addaction name="actBy_Image_fragment"/>
    </widget>
    <addaction name="menu_Search"/>
   </widget>
   <widget class="QMenu" name="menu3D_Models">
    <property name="title">
     <string>3D Models</string>
    </property>
    <addaction name="actView3DMod"/>
    <addaction name="actGenerateGol"/>
   </widget>
   <widget class="QMenu" name="menu_Parameters">
    <property name="title">
     <string>&amp;Parameters</string>
    </property>
    <addaction name="actSettings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actUser_Manual"/>
   </widget>
   <addaction name="menu_DB"/>
   <addaction name="menu_RLI_Analisys"/>
   <addaction name="menu3D_Models"/>
   <addaction name="menu_Parameters"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="act3D_Models">
   <property name="text">
    <string>3D &amp;Models</string>
   </property>
  </action>
  <action name="actRLI">
   <property name="text">
    <string>&amp;RLI</string>
   </property>
  </action>
  <action name="actE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actComparison">
   <property name="text">
    <string>&amp;Comparison</string>
   </property>
  </action>
  <action name="actTestPlugin">
   <property name="text">
    <string>test plugin</string>
   </property>
  </action>
  <action name="actPlugins">
   <property name="text">
    <string>&amp;Plugins ...</string>
   </property>
  </action>
  <action name="actSearchImage">
   <property name="text">
    <string>by &amp;Image</string>
   </property>
  </action>
  <action name="actGologram">
   <property name="text">
    <string>by &amp;Gologram</string>
   </property>
  </action>
  <action name="actView_real_image">
   <property name="text">
    <string>View real image</string>
   </property>
  </action>
  <action name="actBy_Image_fragment">
   <property name="text">
    <string>by RLI fragment</string>
   </property>
  </action>
  <action name="actUser_Manual">
   <property name="text">
    <string>User Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actView3DMod">
   <property name="text">
    <string>View 3D Model</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actGenerateGol">
   <property name="text">
    <string>Generate &amp;gologram</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
